source/Projectile.cpp

```diff
@@ -433,7 +435,7 @@ void Projectile::CheckLock(const Ship &target)
 		double multiplier = 1.;
 		if(distance <= shortRange)
 			multiplier = 2. - distance / shortRange;
-		double probability = weapon->InfraredTracking() * min(1., target.Heat() * multiplier + .05);
+		double probability = weapon->InfraredTracking() * max(0., min(1., (target.Heat() - .05) * multiplier));
```

Comment:
```suggestion
		double probability = weapon->InfraredTracking() * clamp((target.Heat() - .05) * multiplier, 0., 1.);
```
But with this PR's formula, if you have full heat (=1), you'll get 0.95 probability?

Reply:
Right, I need to multiply by (1/0.95).

Reply:
couldn't we rather have something like `probability = weapon->InfraredTracking() * (target.Heat() > 0.05 ? target.Heat() : 0.0)` instead of weird tiny multipliers? Or just directly using `target.Heat()`

Reply:
Your first suggestion would cause the tracking probability to jump up to 5% at 5% heat, rather than starting from 0% at 5% heat and then rising from there.

The second would lose the no tracking at <=5% heat.

Reply:
5% is indeed arbitrary.
My reasoning for wanting a cutoff such that there is zero tracking at and below a low but non-zero temperature is based on something I vaguely remember about heat never actually being at zero when it's checked, because cooling can't bring it below zero and the tracking happens after generating heat but before cooling is applied.
I can't work out whether that is actually the case from looking at the code, but if it is indeed true then that would mean you wouldn't be able to get to 0% tracking even at 0% heat, which would be a problem.

Reply:
Dropping the 5% thing and just using the value directly seems worth trying, at least. We can come back to it if it turns out to be problematic.

Reply:
```suggestion
		double probability = weapon->InfraredTracking() * min(1., target.Heat() * multiplier);
```